drivers/memc/Kconfig.stm32

```diff
@@ -3,7 +3,7 @@
 
 config MEMC_STM32
 	bool "Enable STM32 Flexible Memory Controller (FMC)"
-	depends on SOC_SERIES_STM32H7X
+	depends on SOC_SERIES_STM32H7X || SOC_SERIES_STM32F4X
```

Comment:
This should be added in a separate commit

Reply:
Ok. Shall I move it to a new branch and submit a new pull request or force push it?

Reply:
All changes should be made on the same PR so that we can track review progress

Reply:
I think it's all coming in now in the same PR. I know there was an old PR but you have closed it right?